examples/dialog-modal/alertdialog.html

```diff
@@ -68,9 +68,17 @@ <h2 id="ex_label">Example</h2>
         <div id="ex_alertdialog">
           <label for="notes">Notes</label>
           <textarea class="notes" name="notes" id="notes" rows="7">This is an example text box, which unsurprisingly contains text. The user is given the option to save this text - which triggers a basic alert - or to discard the text - which triggers an alert dialog that prompts the user for confirmation.</textarea>
-          <button type="button" aria-controls="notes" id="notes_save">Save</button>
+          <div class="visually-hidden" id="notes_save_status" role="alert"></div>
+          <button type="button" aria-controls="notes" id="notes_save">
```

Comment:
This should not have aria-controls

Reply:
@smhigley will push change in correcting the case - removing aria-control broke the dialog.